articles/openshift/oshift-scaling-references.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Considerations When Scaling an OpenShift Cluster | UKCloud Ltd
+description: Gives information on things to consider when scaling a cluster
+services: openshift
+author: Kieran O'Neill
+toc_rootlink: References
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Considerations When Scaling an OpenShift Cluster
+toc_fullpath: References/oshift-scaling-references.md
+toc_mdlink: oshift-scaling-references.md
+---
+
+# Considerations When Scaling an OpenShift Cluster
+
+### Infrastructure applications
+
+With your UKCloud for OpenShift cluster we deploy a number of infrastructure applications by default. These are quality of life features that are intended to help you with the administration, management and application development within your cluster. Examples are the EFK stack, Hawkular Metrics, the integrated registry, the service catalog and; with v3.11 deployments onwards, Prometheus and Grafana.
```

Comment:
Use a comma instead of a semicolon. The rule is that you use semicolons to separate items in a list if any of the items include commas, so technically it should be the following (but I don't think it's really necessary):

Examples include EFK stack; Hawkular Metrics; the integrated registry; the service catalog and, with v3.11 deployments onwards, Prometheus and Grafana.